src/measure_theory/pi.lean

```diff
@@ -237,7 +237,7 @@ lemma pi_eval_preimage_null {i : ι} {s : set (α i)} (hs : μ i s = 0) :
   measure.pi μ (eval i ⁻¹' s) = 0 :=
 begin
   /- WLOG, `s` is measurable -/
-  rcases exists_is_measurable_superset_of_null hs with ⟨t, hst, htm, hμt⟩,
+  rcases exists_measurable_set_superset_of_null hs with ⟨t, hst, htm, hμt⟩,
```

Comment:
Browsing at random. This name could be improved, I think. Replace all `measurable_set_superset` with `measurable_superset`?

Reply:
I found one more name of the form `measurable_set.*set.*`, fixed.